Dockerfile

```diff
@@ -21,10 +21,7 @@ RUN yum -y install \
 RUN mkdir -p $NVM_DIR \
  && curl -s https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | bash \
  && . $NVM_DIR/nvm.sh \
- && nvm install 10 \
  && nvm install 12 \
```

Comment:
Version 14 is LTS now - should we use that version instead?

Reply:
node 10 is EOL in April 2021 we should bump this at that time imo.